Enable/Utilities/LoadingViewController.m

```diff
@@ -0,0 +1,37 @@
+//
+//  LoadingViewController.m
+//  Enable
+//
+//  Created by Amanda Wang on 7/20/22.
+//
+
+#import "LoadingViewController.h"
+
+@interface LoadingViewController ()
```

Comment:
I don't think this should be it's own view controller, you want add an activity indicator on every view controller loading data, with something that looks like:
* hide subviews
* start/show activity indicator
* make data fetching call
* get data back
* stop/hide activity indicator
* IF error show error
* ELSE inject the data to the view and unhide subviews